lib/sprockets/sass_importer.rb

```diff
@@ -1,2 +1,9 @@
-# Deprecated: Require sprockets/sass_processor instead
-require 'sprockets/sass_processor'
+
+require 'sprockets/autoload'
+
+module Sprockets
+  class SassImporter < Autoload::Sass::Importers::Filesystem
+    # NOTE: Hack to support sourcemap generation in Sass-3.3
+    def public_url(a, b);'';end
```

Comment:
Also, are we not loosing this information for projects using sass 3.4? Or this method doesn't exist anymore?